pkg/apiserver/user/auth.go

```diff
@@ -44,7 +46,7 @@ type AuthService struct {
 }
 
 type authenticateForm struct {
-	IsTiDBAuth bool   `json:"is_tidb_auth" binding:"required"`
+	IsTiDBAuth *bool  `json:"is_tidb_auth" binding:"required"`
```

Comment:
Why does it become a pointer now?

Reply:
ping

Reply:
I found `bool` will always true, if some param passed,  and found a solution to switch to `*bool`  https://github.com/gin-gonic/gin/issues/814